### PR TITLE
 MKC  7 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/final_word_phantom.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Name:Final-Word Phantom
+ManaCost:2 U
+Types:Creature Spirit Detective
+PT:1/4
+K:Flash
+K:Flying
+S:Mode$ CastWithFlash | ActivationPhases$ End of Turn | ValidCard$ Card | ValidSA$ Spell | Caster$ You | Description$ During each opponent's end step, you may cast spells as though they had flash
```
Comment (Simisays) [inline forge-gui/res/cardsfolder/upcoming/final_word_phantom.txt:7]: `ActivationPhases$ End of Turn`  isn't correct. Any ideas what would be correct here ?

File: forge-gui/res/cardsfolder/upcoming/foreboding_steamboat.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
+Name:Foreboding Steamboat
+ManaCost:3 B B
+Types:Artifact Vehicle
+PT:5/7
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigCataclysm | TriggerDescription$ When CARDNAME enters the battlefield, each player chooses two nontoken, non-Vehicle creatures they control. Exile them until CARDNAME leaves the battlefield.
+SVar:TrigCataclysm:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ ChooseArtf | SubAbility$ ExileChosen
+SVar:ChooseArtf:DB$ ChooseCard | Defined$ Remembered | Amount$ 2 | ImprintChosen$ True | Choices$ Creature.nonToken+nonVehicle+RememberedPlayerCtrl | ChoiceTitle$ Choose two nontoken, non-Vehicle creatures you control 
+SVar:ExileChosen:DB$ ChangeZone | Defined$ Imprinted | Origin$ Battlefield | Destination$ Exile | Duration$ UntilHostLeavesPlay
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/foreboding_steamboat.txt:8]: missing cleanup

File: forge-gui/res/cardsfolder/upcoming/havoc_eater.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Name:Havoc Eater
+ManaCost:5 R R
+Types:Creature Elemental
+PT:3/3
+K:Flying
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigGoad | TriggerDescription$ When CARDNAME enters the battlefield, for each opponent, goad up to one target creature that opponent controls. Put X +1/+1 counters on Havoc Eater, where X is the total power of creatures goaded this way.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/havoc_eater.txt:6]: CARDNAME

File: forge-gui/res/cardsfolder/upcoming/havoc_eater.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+SVar:TrigGoad:DB$ Goad | TargetMin$ 0 | SubAbility$ DBPutCounter | TargetMax$ X | ValidTgts$ Creature.OppCtrl | TargetsWithDifferentControllers$ True | RememberGoaded$ True | TgtPrompt$ Goad up to one target creature that opponent controls
+SVar:DBPutCounter:DB$ PutCounter | CounterType$ P1P1 | CounterNum$ Y
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/havoc_eater.txt:8]: missing cleanup

File: forge-gui/res/cardsfolder/upcoming/immortal_obligation.txt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Name:Immortal Obligation
+ManaCost:1 W
+Types:Instant
+A:SP$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | RememberChanged$ True | TgtPrompt$ Select target creature card in an opponent's graveyard | ValidTgts$ Creature.OppOwn | SubAbility$ DBEffect | WithCountersType$ DUTY | SpellDescription$ Return target permanent card in an opponent's graveyard to the battlefield under their control.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/immortal_obligation.txt:4]: wrong description

File: forge-gui/res/cardsfolder/upcoming/immortal_obligation.txt [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+SVar:DBEffect:DB$ Effect | RememberObjects$ Remembered | StaticAbilities$ Static,StaticTwo,StaticThree | ForgetOnMoved$ Battlefield | ForgetCounter$ DUTY | Duration$ Permanent
+SVar:Static:Mode$ Continuous | Affected$ Creature.IsRemembered | Goad$ True | Description$ For as long as that creature has a duty counter on it, it is goaded, can't attack you or a permanent you control, and can't block creatures you control.
+SVar:StaticTwo:Mode$ CantBlockBy | ValidAttacker$ Card.IsRemembered | ValidBlocker$ Creature.YouCtrl | Secondary$ True | For as long as that creature has a duty counter on it, it is goaded, can't attack you or a permanent you control, and can't block creatures you control.
+SVar:StaticThree:Mode$ CantAttack | ValidCard$ Card.IsRemembered | Target$ You | Secondary$ True | Secondary$ True | Description$ For as long as that creature has a duty counter on it, it is goaded, can't attack you or a permanent you control, and can't block creatures you control.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/immortal_obligation.txt:8]: wrong target

File: forge-gui/res/cardsfolder/upcoming/immortal_obligation.txt [hunk 7]
```diff
@@ -0,0 +1,9 @@
+Name:Immortal Obligation
+ManaCost:1 W
+Types:Instant
+A:SP$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | RememberChanged$ True | TgtPrompt$ Select target creature card in an opponent's graveyard | ValidTgts$ Creature.OppOwn | SubAbility$ DBEffect | WithCountersType$ DUTY | SpellDescription$ Return target permanent card in an opponent's graveyard to the battlefield under their control.
+SVar:DBEffect:DB$ Effect | RememberObjects$ Remembered | StaticAbilities$ Static,StaticTwo,StaticThree | ForgetOnMoved$ Battlefield | ForgetCounter$ DUTY | Duration$ Permanent
+SVar:Static:Mode$ Continuous | Affected$ Creature.IsRemembered | Goad$ True | Description$ For as long as that creature has a duty counter on it, it is goaded, can't attack you or a permanent you control, and can't block creatures you control.
+SVar:StaticTwo:Mode$ CantBlockBy | ValidAttacker$ Card.IsRemembered | ValidBlocker$ Creature.YouCtrl | Secondary$ True | For as long as that creature has a duty counter on it, it is goaded, can't attack you or a permanent you control, and can't block creatures you control.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/immortal_obligation.txt:7]: wrong too

File: forge-gui/res/cardsfolder/upcoming/foreboding_steamboat.txt [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigChange | TriggerDescription$ Whenever CARDNAME attacks, put a card exiled with it into its owner's graveyard. If you do, investigate.
+SVar:TrigChange:DB$ ChangeZone | Mandatory$ True | Hidden$ True | Origin$ Exile | Destination$ Graveyard | ChangeType$ Card.ExiledWithSource | ChangeTypeDesc$ card exiled with CARDNAME | ChangeNum$ 1 | RememberChanged$ True | SubAbility$ DBInvestigate
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/foreboding_steamboat.txt:10]: why not use ExiledMoveToGrave cost?

File: forge-gui/res/cardsfolder/upcoming/final_word_phantom.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/final_word_phantom.txt:7]: just Phases$?
you're not activating anything

File: forge-gui/res/cardsfolder/upcoming/immortal_obligation.txt (hunk 7 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/immortal_obligation.txt]: now 50% less wrong

File: forge-gui/res/cardsfolder/upcoming/final_word_phantom.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/final_word_phantom.txt]: `PlayerTurn$ Opponent`

File: forge-gui/res/cardsfolder/upcoming/final_word_phantom.txt [hunk 9]
```diff
@@ -0,0 +1,8 @@
+Name:Final-Word Phantom
+ManaCost:2 U
+Types:Creature Spirit Detective
+PT:1/4
+K:Flash
+K:Flying
+S:Mode$ CastWithFlash | Phases$ End of Turn | PlayerTurn$ Opponent | ValidCard$ Card | ValidSA$ Spell | Caster$ You | Description$ During each opponent's end step, you may cast spells as though they had flash
+Oracle:Flash\nFlying\nDuring each opponent's end step, you may cast spells as though they had flash
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/final_word_phantom.txt:8]: .

**Review by tool4ever — APPROVED** [review]

**Pull request merged**